examples/deform/README.md

```diff
@@ -0,0 +1,10 @@
+# Deformation examples
+
+These examples show how you can use ODL to perform deformation of images. 
+
+## Basic usage examples
+
+Example | Purpose | Complexity
+------- | ------- | ----------
+[`linearized_fixed_displacement.py`](linearized_fixed_displacement.py) | Example using the operator of **fixed-displacement** linearized deformation | low
```

Comment:
We should agree on a style for the "Purpose" column. I think it should start with a verb ("create", "use", "solve", etc.) since that answers the question "What is the purpose of this example?" Here I think it should be something like "Showcase the usage of ..."